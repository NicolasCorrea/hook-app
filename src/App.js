// import CounterApp from "./components/01-useState/CounterApp"
// import CounterWithHook from "./components/01-useState/CounterWithHook";
// import SimpleForm from "./components/02-useEffect/SimpleForm"
// import FormWithCustomHook from "./components/02-useEffect/FormWithCustomHook"
// import { MultipleCustomHooks } from "./components/03-examples/MultipleCustomHooks"
// import { FocusScreen } from "./components/04-useRef/FocusScreen"
// import { RealExampleRef } from "./components/04-useRef/RealExampleRef"
// import { LayoutEffect } from "./components/05-useLayoutEffect/LayoutEffect"
// import { Memorize } from "./components/06-memos/Memorize"
// import { MemoHook } from "./components/06-memos/MemoHook"
// import { CallbackHook } from "./components/06-memos/CallbackHook"

import { Padre } from "./components/07-tarea-memo/Padre"

const App = () => {
  return (
    <>
      {/* <h1>Hello World</h1> */}
      {/* <CounterApp /> */}
      {/* <CounterWithHook /> */}
      {/* <SimpleForm /> */}
      {/* <FormWithCustomHook /> */}
      {/* <MultipleCustomHooks /> */}
      {/* <FocusScreen /> */}
      {/* <RealExampleRef /> */}
      {/* <LayoutEffect /> */}
      {/* <Memorize /> */}
      {/* <MemoHook /> */}
      {/* <CallbackHook /> */}
      <Padre />
    </>
  );
}

export default App;
