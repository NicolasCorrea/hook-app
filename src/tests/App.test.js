import { shallow } from "enzyme"
import App from "../App"

describe('test on App', () => {
  
  test('should render', () => {
      const wrapper = shallow(<App />)

      expect(wrapper).toMatchSnapshot();

  })
  

})
