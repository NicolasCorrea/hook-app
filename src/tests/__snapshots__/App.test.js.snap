// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on App should render 1`] = `
<Fragment>
  <Padre />
</Fragment>
`;
